python/py-maturin/Portfile

```diff
@@ -0,0 +1,342 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           cargo_fetch 1.0
+PortGroup           python 1.0
+
+name                py-maturin
+version             0.12.18
+revision            0
+categories-append   devel
+license             MIT
+platforms           darwin
+supported_archs     arm64 x86_64
+
+python.versions     37 38 39 310
+
+maintainers         nomaintainer
+
+description         Build and publish crates with pyo3, rust-cpython and cffi bindings as well as rust binaries as python packages.
+long_description    {*}${description}
+
+homepage            https://github.com/pyo3/maturin
+
+checksums           ${distname}${extract.suffix} \
+                    rmd160  694d382c774a265beebdcc159dd2d72ccd27c4d1 \
+                    sha256  ea5e7ee199ad03a031a4b2f402a7dbaa46bf24cd2e41088420e86a9c864d7e07 \
+                    size    155782
+
+if {${name} ne ${subport}} {
+    depends_build-append \
+                    port:py${python.version}-setuptools \
+                    port:py${python.version}-setuptools-rust \
+                    port:py${python.version}-typing_extensions
```

Comment:
this dependency was missing from `setuptools-rust` but not listed by upstream here. Since we added that one now to `setuptools-rust` there shouldn't be a need to add it here explicitly again.

Reply:
Thanks removed.